code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -118,3 +118,44 @@
 				/obj/item/weapon/relic = 3,
 				"" = 11
 				)
+
+/obj/effect/spawner/lootdrop/trade_sol_rare
+	name = "trader rare item spawner"
+	lootdoubles = 0
+	color = "#00FFFF"
+
+	loot = list(
+				/obj/item/weapon/card/emag_broken = 2,
+				/obj/item/weapon/defibrillator/compact/combat/loaded = 1,
+				/obj/item/weapon/defibrillator/compact/loaded = 2,
+				/obj/item/weapon/gun/energy/laser/retro = 2,
+				/obj/item/weapon/rcd/combat = 1,
+				/obj/item/weapon/rcd = 2,
+				/obj/item/weapon/pinpointer/advpinpointer = 3,
+				/obj/item/weapon/gun/medbeam = 2,
```

Comment:
No. No mediguns, or nuke ops gear in general, even if the crewmembers have to trade for them. Combat RCDs aren't as bad. You could also toss something like advanced magboots here though.